packages/esm-patient-registration-app/src/patient-registration/field/person-attributes/location-person-attribute-field.resource.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback, useMemo } from 'react';
+import { type FetchResponse, fhirBaseUrl, openmrsFetch, useDebounce } from '@openmrs/esm-framework';
+import { type LocationEntry, type LocationResponse } from '@openmrs/esm-service-queues-app/src/types';
+import useSWR from 'swr';
+interface IUseLocations {
```

Comment:
I don't believe this adheres to our conventional coding standards for interfaces.  Don't prefix with `I` as a starter, not sure about the verbage in general...

Reply:
```suggestion
interface UseLocationsResult {
```